### ISSUE TITLE
Select a compliant design for callout boxes

### ISSUE THREAD

**Issue by crwallace** [department-of-veterans-affairs/vets-website#2681]
@melwoodard -- Would love you input on this.

**Comment by melwoodard** [comment]
@crwallace is there a specific 508 compliance issue for these? Is it the contrast of the light blue background?

**Comment by crwallace** [comment]
@melwoodard -- the issue is that when the callout boxes have links, the contrast is not strong enough and then compliance fails.

**Comment by Drakenhart** [comment]
@crwallace @melwoodard If bolding it doesn't work, maybe make it the darker blue when in a call out box? Or if that is not doable, maybe change it to a darker gray? These are colors you already have in your palette.

If that doesn't work maybe consider adding one more color to the palette as a secondary link color?

**Comment by jcmeloni** [comment]
@melwoodard  confirming that there is still a decision to be made on this?

**Comment by melwoodard** [comment]
Yes, thanks!

@crwallace we only need to go a smidge darker to `#004795` for the links to pass compliance. The visual difference is almost imperceptible, so I say we just modify the global link color rather than make a special case for the blue callouts.
